client/src/components/CreateOrUpdateForm/CreateOrUpdateForm.js

```diff
@@ -1025,49 +1030,66 @@ export default class CreateOrUpdateForm extends React.Component {
                     </Form.Group>
 
                     <div className="address-coordinates-container">
-                        <span className="font-weight-bold">Please specify one of the following:</span>
-
-                        {/* Address */}
-                        <Form.Group controlId="create-form-address">
-                            <Form.Label>Address:</Form.Label>
-                            <Form.Control
-                                type="text"
-                                name="address"
-                                placeholder="Address"
-                                value={address.value}
-                                onChange={(event) => this.handleInputChange(event)}
-                                isInvalid={!address.isValid}
-                                className="text-control"
-                            />
-                            <Form.Control.Feedback type="invalid">{address.message}</Form.Control.Feedback>
+                        <span className="font-weight-bold">Location Type:</span>
```

Comment:
Minor: Just noticed this while testing out the reducers...I think that this `<span>` can have a little less top padding/margin/whatever it has